### DISCUSSION TITLE
Claimblock retrieval upon world deletion

### DISCUSSION THREAD

**Discussion by LogGits** [GriefPrevention/GriefPrevention#1938] [General]
i need a way to give players all their claimblocks (we had claiming in the end world and we're deleting it now, in doing so will players automatically get their claimblocks back for the deleted world?)

How should i go about handling this situation?

**Comment by Jikoo** [comment]
Available claim blocks are total claim blocks - claim blocks consumed by claims, so yes, if you delete a world's claims players will have access to any blocks they may have used creating claims in it. There's a `/deleteworldclaims` or similar I believe.
